hazelcast/src/test/java/com/hazelcast/config/ClientYamlSchemaTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import org.json.JSONObject;
+import org.junit.runners.Parameterized;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+public class ClientYamlSchemaTest
```

Comment:
Are these tests picked up by PR builder ? Or do we need tp put test annotations to these classes as well (like @Category etc) ?

Reply:
the superclass `AbstractYamlSchemaTest` does have the necessary annotations, so I'm pretty sure it works, but I will look at the build logs once this currently running build finished.

Reply:
Yes, they run on CI: http://jenkins.hazelcast.com/job/Hazelcast-pr-builder/5212/testReport/com.hazelcast.config/ClientYamlSchemaTest/
thanks for the approval, merging the PR.